tool/main.cpp

```diff
@@ -12,6 +12,8 @@
 #include <filesystem>
 #include <fstream>
 #include <iostream>
+#include <spdlog/common.h>
```

Comment:
Have to include both?

Reply:
Actually, no, this is just clangd's work :angry: